user_guide_src/source/incoming/incomingrequest/13.php

```diff
@@ -5,4 +5,4 @@
 [
     'Param1' => 'Value1',
     'Param2' => 'Value2'
-]
+];
```

Comment:
This is the output of `var_dump()`.
The `;` at the end is odd.

What if we made the array a comment?

Reply:
Oh, you're right.
Comment seems reasonable in that case. 
I think there were more like this,  will check.